### ISSUE TITLE
Bug: When sending a command PC via Home Assistant, restart is initiated.

### ISSUE THREAD

**Issue by mxp83** [LAB02-Research/HASS.Agent#307]
So I have a custom command set in Hass Agent, which presses some hotkeys via batch file to switch my main monitor.  Whenever I activate it in Home Assistant, it gives me a prompt that the computer will be restarted (it gives me about a minute before it does).  Now in the commands list I also have 2 commands above my monitor switch, Shutdown and Restart respectively.  Are those commands triggering?

The hotkey command runs a program called Display Fusion, purchased off Steam.  Manually pressing the hotkeys on my pc doesn't replicate the problem and runs normally.  The batch file also has a call for nircmdc, which doesn't work but I understand it requires SendKey format.

Here is the batch code:

> @echo off
> 
> //Switch to TV
> nircmdc sendkeypress 0x10+0x12+0x76
> 
> nircmdc cmdwait 6000
>
> 
> START C:\Playnite\Playnite.FullscreenApp.exe

Below is the Commands Config window in HASS.agent:

![hass-error](https://user-images.githubusercontent.com/14003174/235876372-58b3635e-3e85-4ab6-8b26-2d97b89b9752.jpg)

I wish I could show the Shutdown warning screenshot but I don't get much time to save the image once it occurs.

Current Setup:
Windows 11
Hass Agent 2022.14.0 - installer file used.
Home Assistant 2023.4.6
Supervisor 2023.04.1
Operating System 10.1
Frontend 20230411.1 - latest
Display Fusion Pro 10.0 - installed via Steam

**Comment by mcorsillo** [comment]
> nircmdc sendkeypress 0x10+0x12+0x76

this corresponds to `shift+alt+F7`, per [the nircmd documentation](https://www.nirsoft.net/utils/nircmd2.html#using) and [the linked keycode list](https://learn.microsoft.com/en-us/windows/win32/inputdev/virtual-key-codes?redirectedfrom=MSDN). is this what you're trying to trigger?

i think what's more relevant is what call HomeAssistant is making to the Hass Agent. can you share that call @mxp83?

**Comment by mxp83** [comment]
> this corresponds to shift+alt+F7, per [the nircmd documentation](https://www.nirsoft.net/utils/nircmd2.html#using) and [the linked keycode list](https://learn.microsoft.com/en-us/windows/win32/inputdev/virtual-key-codes?redirectedfrom=MSDN). is this what you're trying to trigger?

That is correct.  This set of keys changes my monitor to a different one based on the setting I put in displayfusion.

Fiddling around previously I discovered that Hass Agent was running all the commands consecutively, so in the image that Playnite batch file was 3rd, it would actually run the 4th and loop backup to the 1st command.  I circumvented this by deleting the restart and shutdown commands and just use the Satellite version.  Though I would like to know why it still looped up in the first place.

Unfortunately I am not near my PC for a while and I can't send the batch file for viewing but will try to remember the post when I return from my trip.
